configs/10.0/packages/mpfr/sources

```diff
@@ -28,7 +28,7 @@ ATSRC_PACKAGE_RELFIXES=
 ATSRC_PACKAGE_VERREV=${ATSRC_PACKAGE_VER}${ATSRC_PACKAGE_REV:+-${ATSRC_PACKAGE_REV}}
 ATSRC_PACKAGE_STR_VER="${ATSRC_PACKAGE_NAME} ${ATSRC_PACKAGE_VERREV}"
 ATSRC_PACKAGE_PRE="test -d mpfr-${ATSRC_PACKAGE_VERREV}"
-ATSRC_PACKAGE_CO=([0]="svn co --revision ${ATSRC_PACKAGE_REV} svn://scm.gforge.inria.fr/svn/mpfr/branches/${ATSRC_PACKAGE_VER%.*} mpfr-${ATSRC_PACKAGE_VERREV}")
+ATSRC_PACKAGE_CO=([0]="svn co --revision ${ATSRC_PACKAGE_REV} https://scm.gforge.inria.fr/anonscm/svn/mpfr/branches/${ATSRC_PACKAGE_VER%.*} mpfr-${ATSRC_PACKAGE_VERREV}")
```

Comment:
This seems to be a personal mirror, it's better to add it as an option instead of replacing the official one.